sklearn/neighbors/approximate.py

```diff
@@ -93,7 +93,7 @@ class GaussianRandomProjectionHash(ProjectionToHashMixin,
                                    GaussianRandomProjection):
     """Use GaussianRandomProjection to produce a cosine LSH fingerprint"""
     def __init__(self,
-                 n_components=8,
+                 n_components=32,
```

Comment:
This class is never instantiated with default parameters and the code doesn't run with n_compenents != 32